packages/shared/src/components/Feed.tsx

```diff
@@ -222,7 +222,8 @@ export default function Feed<T>({
     column: number,
   ): void => {
     document.body.classList.add('overflow-hidden');
-    document.body.classList.add('pr-2');
+    document.body.classList.add('pr-0');
+    document.body.classList.add('laptop:pr-2');
```

Comment:
`classList.add` supports multiple arguments as far as I know.
`document.body.classList.add('classA', 'classB', ...)`
I think remove supports it as well. Please confirm

Reply:
Oh, nice. It worked! Thank you! 🙇